pennylane/ops/op_math/linear_combination.py

```diff
@@ -30,71 +30,75 @@
-    >>> H = qml.ops.LinearCombination(coeffs, obs)
-    >>> print(H)
-    0.2 * (X(0) @ Z(1)) + -0.543 * (Z(0) @ Hadamard(wires=[2]))
-
-
-    A ``LinearCombination`` can store information on which commuting observables should be measured together in
-    a circuit:
-
-    >>> obs = [qml.X(0), qml.X(1), qml.Z(0)]
-    >>> coeffs = np.array([1., 2., 3.])
-    >>> H = qml.ops.LinearCombination(coeffs, obs, grouping_type='qwc')
-    >>> H.grouping_indices
-    ((0, 1), (2,))
-
-    This attribute can be used to compute groups of coefficients and observables:
-
-    >>> grouped_coeffs = [coeffs[list(indices)] for indices in H.grouping_indices]
-    >>> grouped_obs = [[H.ops[i] for i in indices] for indices in H.grouping_indices]
-    >>> grouped_coeffs
-    [array([1., 2.]), array([3.])]
-    >>> grouped_obs
-    [[X(0), X(1)], [Z(0)]]
-
-    Devices that evaluate a ``LinearCombination`` expectation by splitting it into its local observables can
-    use this information to reduce the number of circuits evaluated.
-
-    Note that one can compute the ``grouping_indices`` for an already initialized ``LinearCombination`` by
-    using the :func:`compute_grouping <pennylane.ops.LinearCombination.compute_grouping>` method.
+     The ``LinearCombination`` is represented as a linear combination of other operators, e.g.,
+     :math:`\sum_{k=0}^{N-1} c_k O_k`, where the :math:`c_k` are trainable parameters.
+
+     Args:
+         coeffs (tensor_like): coefficients of the ``LinearCombination`` expression
+         observables (Iterable[Observable]): observables in the ``LinearCombination`` expression, of same length as ``coeffs``
+         simplify (bool): Specifies whether the ``LinearCombination`` is simplified upon initialization
+                          (like-terms are combined). The default value is `False`. Note that ``coeffs`` cannot
+                          be differentiated when using the ``'torch'`` interface and ``simplify=True``. Use of this argument is deprecated.
+         grouping_type (str): If not ``None``, compute and store information on how to group commuting
+             observables upon initialization. This information may be accessed when a :class:`~.QNode` containing this
+             ``LinearCombination`` is executed on devices. The string refers to the type of binary relation between Pauli words.
+             Can be ``'qwc'`` (qubit-wise commuting), ``'commuting'``, or ``'anticommuting'``.
+         method (str): The graph colouring heuristic to use in solving minimum clique cover for grouping, which
+             can be ``'lf'`` (Largest First), ``'rlf'`` (Recursive Largest First), `dsatur` (DSATUR), or `gis` (IndependentSet).
+             Defaults to ``'lf'``. Ignored if ``grouping_type=None``.
+         id (str): name to be assigned to this ``LinearCombination`` instance
+
+    .. seealso:: `rustworkx.ColoringStrategy <https://www.rustworkx.org/apiref/rustworkx.ColoringStrategy.html#coloringstrategy>`_
```

Comment:
@EmilianoG-byte, looking at the [PR docs](https://xanaduai-pennylane--6043.com.readthedocs.build/en/6043/code/api/pennylane.ops.op_math.LinearCombination.html?highlight=linearcombination#pennylane.ops.op_math.LinearCombination), there is an over-indent for these lines, and maybe a few below too.

Reply:
I'll take a look at the updated docs once this is fixed. Thanks!

Reply:
Should be good now @trbromley :)